packages/core/src/GenericBanner/GenericBanner.stories.js

```diff
@@ -217,6 +217,30 @@ export const bannerWithURLProps = () => (
   />
 )
 
+export const bannerWithURLPropsAndCustomHeadingSize = () => (
+  <GenericBanner
+    p={2}
+    heading={
+      <Text.span fontWeight='bold' textColor='primary.base' mr={1}>
```

Comment:
does adding a fontSize onto this Text component here achieve the same thing or does the GenericBanner's fontSize prop overwrite it?

Reply:
no, it gets overwritten because the fontSize prop in GenericBanner is being passed in to heading like this : 
`{
        !!heading &&
          React.cloneElement(heading, {
            fontSize: headingFontSize || fontSize,
          })
      }	 `

Reply:
so by giving it another option with headingFontSize is the best non breaking change I came up with haha

Reply:
Ah okay that makes sense. Something I did for the Menu component might work here and it saves us adding another prop `!!heading && React.cloneElement(heading, { fontSize: heading.props.fontSize || fontSize }) }`

MenuList example: https://github.com/priceline/design-system/blob/master/packages/menu/src/MenuList/MenuList.js#L87

Reply:
I don't really like doing `heading.props.fontSize` as it just feels a bit weird to me but that might just be because you don't see this very often 🤷